apps/shared/avifutil.h

```diff
@@ -6,6 +6,12 @@
 
 #include "avif/avif.h"
 
+#ifdef _WIN32
```

Comment:
```suggestion
#ifdef _MSVC || !__USE_MINGW_ANSI_STDIO
```

perhaps this might be better to keep using `%zu` for the printfs that do support it while keeping the change correct for those with `__USE_MINGW_ANSI_STDIO=0`

Reply:
it probably would also help with
```c
C:\projects\libavif\apps\shared\avifutil.c(22,12): error C2220: the following warning is treated as an error [C:\projects\libavif\build\avifdec.vcxproj]
C:\projects\libavif\apps\shared\avifutil.c(22,12): warning C4777: 'printf' : format string '%Iu' requires an argument of type 'unsigned __int64', but variadic argument 2 has type 'size_t' [C:\projects\libavif\build\avifdec.vcxproj]
C:\projects\libavif\apps\shared\avifutil.c(22,12): message : the sizes of types 'size_t' and 'unsigned __int64' might differ on other platforms [C:\projects\libavif\build\avifdec.vcxproj]
C:\projects\libavif\apps\shared\avifutil.c(22,12): message : consider using '%zu' in the format string [C:\projects\libavif\build\avifdec.vcxproj]
C:\projects\libavif\apps\shared\avifutil.c(23,12): warning C4777: 'printf' : format string '%Iu' requires an argument of type 'unsigned __int64', but variadic argument 2 has type 'size_t' [C:\projects\libavif\build\avifdec.vcxproj]
C:\projects\libavif\apps\shared\avifutil.c(23,12): message : the sizes of types 'size_t' and 'unsigned __int64' might differ on other platforms [C:\projects\libavif\build\avifdec.vcxproj]
C:\projects\libavif\apps\shared\avifutil.c(23,12): message : consider using '%zu' in the format string [C:\projects\libavif\build\avifdec.vcxproj]
C:\projects\libavif\apps\shared\avifutil.c(24,12): warning C4777: 'printf' : format string '%Iu' requires an argument of type 'unsigned __int64', but variadic argument 2 has type 'size_t' [C:\projects\libavif\build\avifdec.vcxproj]
C:\projects\libavif\apps\shared\avifutil.c(24,12): message : the sizes of types 'size_t' and 'unsigned __int64' might differ on other platforms [C:\projects\libavif\build\avifdec.vcxproj]
C:\projects\libavif\apps\shared\avifutil.c(24,12): message : consider using '%zu' in the format string [C:\projects\libavif\build\avifdec.vcxproj]
```

Reply:
or /wd4777 ?

Reply:
honestly, i don't see the point in adding /WX and plenty of /wd after just for the sake of not having wrning messages, instead of fixing warning

Reply:
I am assuming that @joedrago found a lot of warnings produced code that is not his, that led to the /wd s after, but in this case, there is a "fix" for this specific warning at least instead of just slapping `/wd4777` into the cmakelist